src/General/Alert/Alert.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react';
+
+import { Alert, Props } from './Alert';
+import { Button } from '../Button';
+import { BaseContainer } from '../../Layout/GlintsContainer/GlintsContainer';
+
+export default {
+  title: 'General/Alert',
+  component: Alert,
+  decorators: [Story => <BaseContainer>{Story()}</BaseContainer>],
+  argTypes: {
+    type: {
+      control: {
+        type: 'select',
+        options: ['success', 'warning', 'danger', 'info'],
```

Comment:
I think we can use intuitive control type (radio button) like 
![image](https://user-images.githubusercontent.com/13249748/101116936-f6a82480-3620-11eb-936a-3d554553f491.png)